.github/workflows/translation-review.yml

```
@@ -0,0 +1,82 @@
+    inputs:
+      pr_number:
+        description: 'PR number to review (manual rerun)'
+        required: true
+        type: number
+      model:
+        description: 'Anthropic model to use (overrides default)'
+        required: false
+        type: choice
+        default: ''
+        options:
+          - ''
```

actionlint (1.7.12)

[error] string should not be empty

(syntax-check)